pkg/util/api.go

```diff
@@ -31,6 +31,7 @@ var (
 	reservedSymbolsRFC3986 = strings.Join([]string{
 		":", "/", "?", "#", "[", "]", "@", "!", "$", "&", "'", "(", ")", "*", "+", ",", ";", "=",
 	}, "")
+	supportedContentTypes = []string{"application/json", "application/x-www-form-urlencoded"}
```

Comment:
Maybe map of supported types strings and boolean? I think it will be easier to use it. Just a thought

Reply:
That's not a bad idea

Reply:
Actually, in this context, this won't be very pragmatic, as the actual content type header will contain additional information in the string as well, ex:

`application/json; charset=utf-8`

So we can't define a map beforehand in which we'll later search with specific request content-type header values. That's why the current implementation uses `strings.Contains` and has to traverse the whole array.